ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/RestApiTest.java

```diff
@@ -289,6 +291,28 @@ public void shouldExecuteServerMetadataIdRequest() {
     assertThat(response, is(notNullValue()));
   }
 
+  @Test
+  public void shouldExecuteAllConfigsRequest() {
```

Comment:
we should add tests to make sure that we don't accidentally leak any config that isn't in the allowed list of configs